content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,468 @@
+      within the **15th day** of the month following the delivery covered by the document.
+
+      It usually is a **summary invoice** containing a list of multiple sales of goods or services,
+      carried out in the month. The business is allowed to **group** the sales into **one invoice**,
+      generally issued at the **end of the month** for accounting purposes. Deferred invoices are
+      default for **wholesaler** having recurrent clients.
+
+      If the goods are transported by a **carrier**, every delivery has an associated **Documento di
+      Transporto (DDT)**, or **Transport Document**. The deferred invoice **must** indicate the
+      details of all the **DDTs** information for better tracing.
 
-You can check the current status of your customer invoice under the **Electronic invoicing** field.
+      .. note::
+         e-Invoicing of deferred invoices requires the `l10n_it_stock_ddt`
+         :ref:`module <italy/modules>`. In this case, a dedicated :guilabel:`Tipo Documento` `TD24`
+         is used in the e-Invoice.
 
-.. image:: italy/processing.png
+      Odoo exports moves as `TD24` if the following conditions are met:
+
+      - Is an invoice;
+      - Is associated to deliveries whose **DDTs** have a **different** date than the issuance date
+        of the invoice.
+
+Vendor bills
+~~~~~~~~~~~~
+
+Italian companies buying goods or services from EU countries (or services from non-EU countries)
+must send the information contained within the bill received to the **Agenzia delle Entrate**. This
+allows you to complete tax-related information on your bill, and to send it. The seller must be set
+as :guilabel:`Cedente/Prestatore`, and the buyer as :guilabel:`Cessionario/Committente`. Contained
+within the **XML** document for the vendor bill, the vendor's credentials show as
+:guilabel:`Cedente/Prestatore`, and your company's credentials as
+:guilabel:`Cessionario/Committente`.
+
+.. note::
+   Self-billing invoices or VAT invoice integrations must be issued and sent to the tax agency.
+
+When inputting taxes in a vendor bill, it is possible to select **reverse charge** taxes. These are
+automatically activated in the Italian fiscal position. By going to :menuselection:`Accounting -->
+Configuration --> Taxes`, the `10%` and `22%` :guilabel:`Goods` and :guilabel:`Services` tax scopes
+are activated and preconfigured with the correct tax grids. These are set up automatically to ensure
+the correct booking of accounting entries and display of the tax report.
+
+For **vendor bills**, **three** types of configurations are technically identified by a code called
+:guilabel:`Tipo Documento`:
+
+- `TD17` - Buying services from **EU** and **non-EU** countries;
+- `TD18` - Buying **goods** from **EU**;
+- `TD19` - Buying **goods** from a **foreign** vendor, but the **goods** are already in **Italy**
+  in a **VAT deposit**.
+
+.. tabs::
+
+   .. tab:: `TD17`
```

Comment:
There is some strange technical problem here that doesn't happen on other tabs in this very same page...
If you click the `TD17` tab, then `TD18` then `TD17` back again, the tab disappears 😅 
Can you please check whether it's your documentation in particular or a bug in the framework?
![image](https://user-images.githubusercontent.com/1665365/215099355-d9aa0b3f-4734-4e74-bd95-0c3402737d3f.png)

Reply:
I guess it is a bug in the framework, since I tried writing things, having it empty, etc... No matter the content of the tab, it was broken. I'm not sure what other methods I can try to test this out.